pkg/image/factory.go

```diff
@@ -113,6 +121,23 @@ func (f *Factory) makeEnvVars() ([]corev1.EnvVar, error) {
 	return envVars, nil
 }
 
+func (f *Factory) makeCacheSize() (*resource.Quantity, error) {
+	if f.CacheSize == "" {
+		return nil, nil
+	}
+
+	c, err := resource.ParseQuantity(f.CacheSize)
+	if err != nil {
+		return nil, errors.New("invalid cache size, must be valid quantity ex. 2G")
```

Comment:
Should we wrap this error or we trying to prevent the user from seeing it?

Reply:
@matthewmcnew I thought to not expose it, it looks like this:

```
invalid cache size: quantities must match the regular expression '^([+-]?[0-9.]+)([eEinumkKMGTP]*[-+]?[0-9]*)$'
```